src/main/java/at/hannibal2/skyhanni/features/inventory/ShiftClickNpcSell.kt

```diff
@@ -17,7 +17,7 @@ object ShiftClickNpcSell {
 
     private val config get() = SkyHanniMod.feature.inventory.shiftClickNPCSell
 
-    private val sellSlot = -4
+    private const val sellSlot = -4
```

Comment:
```suggestion
    private const val SELL_SLOT = -4
```
make sure to rename it where it is used as well